svg/Cargo.toml

```diff
@@ -18,5 +18,6 @@ lyon_path_iterator = { version = "0.5.0", path = "../path_iterator" }
 lyon_path_builder = { version = "0.5.0", path = "../path_builder" }
 lyon_path = { version = "0.5.0", path = "../path" }
 lyon_core = { version = "0.5.0", path = "../core" }
+lyon_extra = { version = "0.5.0", path = "../extra" }
```

Comment:
Here I would prefer that you remove the `extern crate lyon_extra;` from svg/src/lib.rs rather than fix the dependency in cargo since the svg crate does not actually use lyon_extra.